backend/src/environment.ts

```diff
@@ -26,15 +25,13 @@ export class Environment {
   isPiZero: boolean
   simpleOS: Ios
   osVersion: string
-  platformCode: number
-  manufactureId: number
   privateIP: ipAddress = ''
   adminUsername: string = ''
   userPath: string
   adminPath: string
   binPath: string
   deprecatedBinaries: string[]
-  manufacturerDetails: IManufacturer
+  manufacturerDetails: ManufacturerDetails
```

Comment:
Having this used directly so that you have to list environment.manufacturerDetils.product.appCode... makes me think it we should at least rename it to just `manufacturer`... also just confirming `appCode` is correct? It's different than at the end of our talk today.

Reply:
We also have to call environment.manufacturerDetils.manufactuer.id. So I think its confusing if we use manufacturer twice.  AppCode is just being reported up to analytics.  It is not being read in from the file.

Reply:
Okay but it is part of the file schema, and it should be read from the file and used in tracking for the remoteitpi image. If it's not the right name and it looks like it's not being allowed to be overriden in the code, then we won't get the right data reported to connectd.